pubspec.yaml

```diff
@@ -109,6 +110,10 @@ dependency_overrides:
   intl: ^0.18.1
   path: ^1.8.3
   test_api: <0.5.0 # Requires Flutter 3.10
+  fimber: ^0.6.6
```

Comment:
We already declare fimber plugins under `dependencies:`

Reply:
I had to override them in order to get the logs for iOS. Does the logging work in production with the latest fimber version.

Reply:
Done